### ISSUE TITLE
Homepage Results/Sort by/Explainer - Designs for Review

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#838]
@scottofletcher @plscully @dethe Here are a couple of options for the homepage results. 

Generally, how do we feel about listing "results" instead of "pages"?

Specifically: 

Option 1 shows our usual treatment, but adds "Sort by" to the far right. Also adds "First, Previous, Next, Last" text with arrows. Currently there's no way to skip to the end of the results, not sure if this matters - thoughts?

Option 2 adds an explainer under the "all" tab (as discussed in this issue: https://github.com/participedia/usersnaps/issues/182). Also shows the "Sort by" menu open and positioned above the line, on the same hierarchical level as cases/methods/orgs (open menu has same visual treatment as clicking your avatar and seeing profile/sign out options). @scottofletcher Are these the options we want to sort by: Date submitted & Last edited ?

Option 3 shows the pagination at the bottom left after the results, back to an earlier design that was never implemented. 

OPTION 1
![home - results - sort by 1-100](https://user-images.githubusercontent.com/13560530/41318103-1b94142e-6e4c-11e8-8063-b50106de81f5.jpg)

OPTION 2
![home - results - sort by 2-100](https://user-images.githubusercontent.com/13560530/41318104-1ba83b5c-6e4c-11e8-9b11-377ca32a84be.jpg)

OPTION 3
![home - results - sort by 3-100](https://user-images.githubusercontent.com/13560530/41318105-1bbea31a-6e4c-11e8-89d8-dae4f6ee11d1.jpg)

**Comment by plscully** [comment]
@jesicarson Thank you, this is very helpful! I like Option 2 the best, although the explainer doesn't look as nice as the rest of the page. I think we need something like this, but it looks a bit squeezed in. And my sense is that it is not only because Option 2 needs two lines to include it.

**Comment by jesicarson** [comment]
Good point, thanks @plscully. I'll keep working on how the explainer can look. I'm not convinced it fits in that position after all, as we also display text results of searches there, ie, in the space below the tabs, [this search](https://participedia.xyz/?query=democ) displays  :  1 result of 1 for "democ" (page 1 of 1)

Also thinking we can get away with just arrows <<  <  >  >>, rather than the words "first, previous, next, last" and a sort button maybe (taking cues from Google):

![screen shot 2018-06-14 at 8 00 12 am](https://user-images.githubusercontent.com/13560530/41420324-06976a40-6fa9-11e8-88f0-28661704391a.png)

**Comment by scottofletcher** [comment]
I like Option 2 as well but I still think we need to reword the 'explainer' to fit with our "new" approach -- include words/phrases like 'wiki', 'crowdsourced', 'community created content', etc. I'm not sure the "stories" really fits here anymore since, like Wikipedia, we (ideally) want descriptive accounts with secondary/academic sources to back up the information. Our guidelines make this pretty clear and I think Pat and I have been pretty consistent in including it in our communiques with collaborators/coapplicants/RAs

**Comment by jesicarson** [comment]
I think the conversation about language, tying into overall vision for Participedia is important and should be continued. Lets find a way to do that... Maybe a focused discussion at the next all team meeting? 

But for now I'm just going to focus on the design side :)

Another couple versions (i guess this would be options 4-7) showing the explainer for "all" and "case" above the tabs, in the same type size as headings in the case narrative. Its a lot of real estate, but maybe important as that was some of the most emphatic feedback - what is this thing??!! 

OPTION 4 (all)
![home - explaner 1 - all- option 4](https://user-images.githubusercontent.com/13560530/41433573-39d94ed0-6fce-11e8-9ff2-6e99ee24f3f9.jpg)

OPTION 4 (case)
![home - explaner 1 - case- option 4](https://user-images.githubusercontent.com/13560530/41433583-43127f8a-6fce-11e8-846b-479e3d6b33d0.jpg)

OPTION 5 (case)
![home - explaner 1 - case - option 5](https://user-images.githubusercontent.com/13560530/41433642-67e66ccc-6fce-11e8-9d3b-7d43eb48cf91.jpg)

OPTION 6 (case)
![home - explaner 1 - case - option 6](https://user-images.githubusercontent.com/13560530/41433657-71237654-6fce-11e8-9170-f83abe20a8bf.jpg)

OPTION 7 (case)
![home - explaner 1 - case - option 7](https://user-images.githubusercontent.com/13560530/41433671-7a2172c4-6fce-11e8-93b0-153276c2c6bf.jpg)

**Comment by scottofletcher** [comment]
Pat and I talked I've actually come around the the "A global community...etc." tag-line that you, Paolo, and I made up like over a year ago :)

As for the new mockups 
I like the ones w/explainers, (not necessarily the wording -- we should stick to the wording our tagline uses: public participation) but we can discuss that later
I also like having results rather than pages
I'll leave the nitty-gritty design details up to you and amber (although I'm most partial to option 7 -- what would it look like if the vertical paragraph line was red?)

**Comment by plscully** [comment]
@jesicarson I agree that you should focus on the design and not worry about the text.
- I very much like that we will have an overall explainer as well as one for each category of content. 
- I also like results instead of pages, and the just arrows << < > >> design is also nice
If you want to start using new, briefer text, you use the ones below for the time being. I'm inclined to just use them and see how people respond rather than go through another long process of wordsmithing. The cases, methods, orgs text are abbreviated versions of the text on our "about" page. 
@scottofletcher  If you have different ideas about wording, you & I can work it through on Slack
- (no need to say "PPedia is a ...") Global Collaboration to Map New Forms of Participatory Politics and Governance
- Cases are events of participatory politics and governance of all shapes and sizes (Delete “and instances”)
- Methods are processes used to guide participatory politics and governance (Delete “the” & “and procedures”)
- Organizations are groups that design, implement, or support innovations in participatory politics and governance (Delete “profiles of formal and informal”)

**Comment by jesicarson** [comment]
Thanks for the feedback, that's great. Will have Amber take a look at the designs next time we chat.

**Comment by jesicarson** [comment]
Thanks @dethe! Good point, I will add pagination to the bottom of the page in the design guidelines #853. 

Regarding directionality of sorting, that would be a @scottofletcher question :)

Regarding design, when Amber reviewed these, she suggested that our feature tour / tool tips #624 would negate the need for permanent explainers. Once it is implemented, you all can test it and see if you agree.

**Comment by jesicarson** [comment]
More requests for a permanent explainer on the homepage. See: https://github.com/participedia/usersnaps/issues/274

I think we should implement option 4, but to keep things simple maybe not change the explainers for case/ method/ org tabs (just leave the main "Participedia is" explainer permanently above the tabs). 

The other thing, @dethe, is that Andrea didn't provide any mockups for the feature tour, so I can't show people what it would be like - do you have it accessible to take a few screenshots by any chance?

**Comment by scottofletcher** [comment]
@plscully not sure about 'participatory politics' - I think "public participation and democratic innovation" better captures what we're about

**Comment by jesicarson** [comment]
The text is in the copy deck (under About and Footer):
https://docs.google.com/document/d/1eUVXSBHN_QqR5O_MCNHpiplAAr51K3EM5yOo8w05I5k/edit#

I would keep it as is, and (if you must) add "and democratic innovation" to
the end:

Participedia: A global community sharing knowledge and stories about public
> participation and democratic innovation


Or, if you're going to change anything, take a cue from Wikipedia and add:
"...that anyone can edit"  :P

*JESI CARSON* BDes - IxD, MDes
*Graphics, Spaces, Interaction*
604-729-9133
Portfolio <http://cargocollective.com/jesicarson> | Facebook
<https://www.facebook.com/jesi.carson> | Twitter
<http://www.twitter.com/jesicarson> | Instagram
<http://www.instagram.com/jesicarson> | Pinterest
<http://www.pinterest.com/jesi> | LinkedIn
<https://ca.linkedin.com/in/jesicarson>

*Design & Communities Coordinator, *Participedia <https://participedia.xyz/>
*Project Lead & Design Thinking Facilitator, *Vancouver Design Nerds
<https://vancouver.designnerds.org>

**Comment by scottofletcher** [comment]
Ok, so we're going to scrap the case/method/tool/technique/org explainers except for those on the About page and in [this doc](https://drive.google.com/drive/folders/0B5j_dGoSFCLJajdfX2RyamdneEk)? 


And actually that last point is good but I still think the "sharing knowledge and stories" leaves out 1) the collaborative nature of the project (ie. I can share _my_ story or _my_ knowledge but it's still mine, I'm the 'owner'), and 2) the documentary aspect (knowledge and stories are _documented_ vs delivered persuasively (as most students/academics are used to)).

**Comment by jesicarson** [comment]
The explainers for all the things are going to be in a few places - on the about page, on the quick submit page and on the FAQs under the help & contact menu. 

Re: tag line wording - I think “community” is important. And “stories” are personal and welcoming. Adding “collaborative” could be interesting. At any rate, why are we thinking of changing it now? So far, people seem to like the wording we all decided on last year, they just want to see it more prominently displayed on the home page.

**Comment by scottofletcher** [comment]
All good points! I'm going to mull this over a bit so we get the right balance. 

The need for change is due to the overwhelming number of people that don't seem to understand what PPedia is. I met numerous people at the NCDD Conference who had gone to our site but didn't understand what it was about. Some people thought cases were scheduled events; others thought we had led the events; others thought they were articles in the academic sense of the term (ie. not collaborative) etc. The second I tell them we're "like Wikipedia but for public participation and democratic innovation" they get it.
